css/src/atomics/background.scss

```diff
@@ -0,0 +1,31 @@
+.background-image-center {
```

Comment:
I think background related classes in docs-ui for now.

Reply:
Although the comment is missing a verb, I think it implies keeping background related classes in docs-ui for now so I removed it.

Reply:
Indeed, sorry you had to infer! 😆